src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -65,9 +74,23 @@ const FluxEditorMonaco: FC<Props> = ({
     connection.current.updatePreludeModel(variables)
   }, [variables])
 
+  useEffect(() => {
+    if (
+      connection.current &&
+      fluxQueryBuilder &&
+      isFlagEnabled('schemaComposition')
```

Comment:
This is the feature flagged turning on of schema composition. This is where the `LspConnectionManager` subscribes to the session store.

Note to @appletreeisyellow , we'll also need to feature flag (using this flag) the turning off of the old style of schema injection.

Reply:
What we **will need to do** is: 
1. have selected fields, tags, and tag_values be stored in the session state
2. update `LspConnectionManager.onSchemaSessionChange()` to detect the diff.
3. upon diff --> we call `LspConnectionManager.inject()`